src/Workleap.OpenApi.MSBuild/SpectralManager.cs

```diff
@@ -22,20 +22,39 @@ public SpectralManager(ILoggerWrapper loggerWrapper, string openApiToolsDirector
     }
     
     private string ExecutablePath { get; set; } = string.Empty;
+    
+    private bool SkipRun { get; set; }
```

Comment:
I believe there's the same issue here, SkipRun isn't necessary, as we will never reached the code where it's bool checked if an exception occurs